assets/js/signup.js

```diff
@@ -1,101 +1,92 @@
-      "lastName": "required",
-      email: {
-        required: true,
-        email: true
-      },
-      phone: {
-        required: true,
-        phoneUS: true
-      }
-    },
-    messages: {
-      email: {
-        required: "We need your email address to confirm your appointment",
-        email: "Your email address must be in the format of name@domain.com"
-      }
-    }
-  });
+	$("#vitaSignupForm").validate({
+		rules: {
+			"firstName": "required",
+			"lastName": "required",
+			email: {
+				required: true,
+				email: true
+			},
+			phone: {
+				required: true,
+				// phoneUS: true
+			}
+		},
+		messages: {
+			email: {
+				required: "We need your email address to confirm your appointment",
+				email: "Your email address must be in the format of name@domain.com"
+			}
+		}
+	});
 }
 
 // Form submission
 $('#vitaSignupForm').submit(function(e) {
-  if (!$(this).valid()) {
-    return false;
-  }
+	// Stop default form submit action
+	e.preventDefault();
 
-  // AJAX Code To Submit Form.
-  $.ajax({
-    url: "/server/signup.php",
-    type: "post",
-    dataType: 'json',
-    data: $(this).serialize(),
-    cache: false,
-    success: function(submitResponse){
-      console.log(submitResponse);
-      window.location = "/";
-    }
-  });
+	if (!$(this).valid()) {
+		return false;
+	}
 
-  return true;
-});
+	var questions = [];
+	$('.vita-form-radio').each(function(){
+		var checkedRadioBox = $(this).find('input[type="radio"]:checked');
+
+		questions.push({
+			id: checkedRadioBox.attr('name'),
+			value: checkedRadioBox.val()
+		});
+	});
+
+	var data = {
+		"firstName":firstName.value,
+		"lastName":lastName.value,
+		"email":email.value,
+		"phone":phone.value,
+		"questions": questions,
+
+		//TODO
+		"scheduledTime": '2017-07-26T15:30:00',
+		"siteId": 4
+	};
 
-// To be deleted at a later point
-function validateSampleForm() {
-  var validationString = '{ "rules": {' +
-  '    "firstName": "required",' +
-  '    "lastName": "required",' +
-  '    "email": {' +
-  '      "required": true,' +
-  '      "email": true' +
-  '    },' +
-  '    "phone": {' +
-  '      "required": true,' +
-  '      "phoneUS": true' +
-  '    },' +
-  '    "address1": "required",' +
-  '    "city": "required",' +
-  '    "state": {' +
-  '      "required": true,' +
-  '      "stateUS": true' +
-  '    },' +
-  '    "zip": {' +
-  '      "required": true,' +
-  '      "zipcodeUS": true' +
-  '    },' +
-  '    "date": {' +
-  '      "required": true,' +
-  '      "date": true' +
-  '    }' +
-  '  },' +
-  '  "messages": {' +
-  '    "email": {' +
-  '      "required": "We need your email address to confirm your appointment",' +
-  '      "email": "Your email address must be in the format of name@domain.com"' +
-  '    }' +
-  '  }' +
-  '}';
+	// AJAX Code To Submit Form.
+	$.ajax({
+		url: "/server/storeAppointment.php",
+		type: "post",
+		dataType: 'json',
+		data: (data),
+		cache: false,
+		complete: function(response){
```

Comment:
Could do that but I'm not sure on what to do on the `error` callback? Tell them to try again with better internet?